frontend/src/components/custom/CustomFilter.tsx

```diff
@@ -26,36 +37,33 @@ const CustomFilter: React.FC<CustomFilterProps> = ({ currentView }) => {
 	const [producerSearchQuery, setProducerSearchQuery] = useState('')
 	const [subscriptionSearchQuery, setSubscriptionSearchQuery] = useState('')
 
-	const filteredCheckboxes = (searchQuery: string, completeArray: string[]) => {
-		let filteredArr = []
-		filteredArr = completeArray.filter((item: string) => {
-			return item.includes(searchQuery)
-		})
-		return filteredArr
-	}
-
-	const filteredClusters = filteredCheckboxes(
```

Comment:
was this deleted code from line 29-37 deprecated or inefficient? And I hope it's working as intended :D

Reply:
I renamed the function to filteredBySearchQuery and moved it into the Helpers.tsx file.